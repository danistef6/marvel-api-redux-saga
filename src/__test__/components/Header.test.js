import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../../components/pages/Header/index';

describe('Header Snapshot', () =>{
    test('Comprobar la UI del componente Header', () => {
        const header = create(<Header />);
        expect(header.toJSON()).toMatchSnapshot();
    });
});