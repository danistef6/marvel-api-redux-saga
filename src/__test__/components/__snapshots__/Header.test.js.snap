// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot Comprobar la UI del componente Header 1`] = `
<nav
  className="sc-bdVaJa dojfSo"
>
  <a
    className="sc-bwzfXH hEEFWg"
  />
</nav>
`;
